asapdiscovery/simulation/scripts/MproP0009.py

```diff
@@ -0,0 +1,170 @@
+# an openforcefield Molecule object that was created from a RDKit molecule.
+# The topology part is described in the openforcefield API but the positions part grabs the first (and only)
+# conformer and passes it to Modeller. It works. Don't ask why!
+modeller.add(ligand_mol.to_topology().to_openmm(), ligand_mol.conformers[0].to_openmm())
+
+# Input Files
+
+from openmm.app import ForceField
+#pdb = PDBFile(args.input_pdb_path)
+forcefield = ForceField('amber14-all.xml', 'amber14/tip3pfb.xml')
+
+# System Configuration
+
+from openmm.app import *
+from openmm.unit import * 
+
+nonbondedMethod = PME
+nonbondedCutoff = 1.0*nanometers
+ewaldErrorTolerance = 0.000001
+constraints = HBonds
+rigidWater = True
+constraintTolerance = 0.000001
+hydrogenMass = 4.0*amu
+
+
+
+# Integration Options
+
+dt = 0.004*picoseconds
+temperature = 300*kelvin
+friction = 1.0/picosecond
+pressure = 1.0*atmospheres
+barostatInterval = 25
+
+from openmm.app import PDBReporter
+from openmm.app import DCDReporter
+from openmm.app import StateDataReporter
+from openmm.app import CheckpointReporter
+from openmm import Platform
+# Simulation Options
+
+steps = 1000000
+equilibrationSteps = 1000
+platform = Platform.getPlatformByName('Reference')
+dcdReporter = DCDReporter(os.path.join(args.output_dir,'trajectory.dcd'), 10000)
+dataReporter = StateDataReporter(os.path.join(args.output_dir,'log.txt'), 1000, totalSteps=steps,
+    step=True, speed=True, progress=True, potentialEnergy=True, temperature=True, separator='\t')
+checkpointReporter = CheckpointReporter(os.path.join(args.output_dir,'checkpoint.chk'), 10000)
+
+# Prepare the Simulation
+from openmm import MonteCarloBarostat
+from openmm import LangevinMiddleIntegrator
+from openmm.app import *
+print('Building system...')
+#topology = pdb.topology
+#positions = pdb.positions
+#modeller = Modeller(pdb.topology, pdb.positions)
+modeller.addSolvent(forcefield, padding=0.9*nanometers, model='tip3p')
+system = forcefield.createSystem(modeller.topology, nonbondedMethod=nonbondedMethod, nonbondedCutoff=nonbondedCutoff,
+    constraints=constraints, rigidWater=rigidWater, ewaldErrorTolerance=ewaldErrorTolerance, hydrogenMass=hydrogenMass)
+system.addForce(MonteCarloBarostat(pressure, temperature, barostatInterval))
+integrator = LangevinMiddleIntegrator(temperature, friction, dt)
+integrator.setConstraintTolerance(constraintTolerance)
+simulation = Simulation(modeller.topology, system, integrator, platform)
+simulation.context.setPositions(modeller.positions)
+
+
+
+# Minimize and Equilibrate
+
+print('Performing energy minimization...')
+simulation.minimizeEnergy()
+print('Equilibrating...')
+simulation.context.setVelocitiesToTemperature(temperature)
+simulation.step(equilibrationSteps)
+
+
+# Simulate
+
+print('Simulating...')
+simulation.reporters.append(PDBReporter(os.path.join(args.output_dir,'output(P0009).pdb'), 10))
+simulation.reporters.append(DCDReporter(os.path.join(args.output_dir,'output(P0009).dcd'), 10))
```

Comment:
Some things:
1. you have already made a dcdreporter in line 124, use that instead of remaking it here!
2. let's change the writing frequency back to a reasonable number (10-100 ps / frame, or whatever Jenke does)
3. We don't want to be using a PDBReporter and a DCDReporter. The PDBReporter will write out the trajectory as a multi-object pdb file, which is wildly data inefficient. DCD / XTC just keeps track of the positions (and velocities? I don't remember) but doesn't keep the topology information, which saves space. Then we can use the original or output PDBFile (on line 168) to load in the topology file for analyzing. 
4. Let's never use parentheses in file names. It's a headache to deal with. For this instead we can just do output_P0009.dcd. In the future we'll want to make this dynamic with something like an "output_prefix" or something so that the `P0009` part of the name isn't hard coded.

Reply:
For some reason (just did another test run), the dcdreporter I created in line 124 is still not receiving any info. Yet the other one ( on line 161) is. I've been playing around with this for a while so still trying to figure it out. My log.txt file is consistently empty as well